dashboard/src/components/ui/sheet.tsx

```diff
@@ -54,12 +54,13 @@ interface SheetContentProps
 const SheetContent = React.forwardRef<
   React.ElementRef<typeof SheetPrimitive.Content>,
   SheetContentProps
->(({ side = "right", className, children, ...props }, ref) => (
+>(({ side = "right", className, children, onKeyDown, ...props }, ref) => (
```

Comment:
Any of this changes are needed, `...props` already pass the `onKeyDown` to the component